lua/litee/symboltree/init.lua

```diff
@@ -342,14 +342,14 @@ local select_config_key = function(key, user_config, default_config)
   local val = user_config[key]
   local default_val = config[key]
 
-  if nil == val then
-    return default_val
-  end
-
   if "table" ~= type(default_val) then
     return val
   end
 
+  if nil == val then
```

Comment:
This is a nit, but the rest of the code flips these comparisons to " var == nil ". Do you mind falling in line and placing variables first when in comparison statements?

Reply:
No problem there.

Reply:
Thanks.